### PR TITLE
DOCSP-45120-change-sync-to-migration

### PR DESCRIPTION
**Pull request by ltran-mdb2**
## DESCRIPTION

Updates mentions of "sync jobs" to "migration jobs"

## STAGING
Because the change is fairly straightforward and involved many pages, I only listed the pages with large changes below:
[Migration Jobs](https://deploy-preview-180--docs-relational-migrator.netlify.app/jobs/sync-jobs/)
[Create a Migration Job](https://deploy-preview-180--docs-relational-migrator.netlify.app/jobs/creating-jobs/)
[Monitor a Migration Job](https://deploy-preview-180--docs-relational-migrator.netlify.app/jobs/monitoring-jobs/)
[Stop a Migration Job](https://deploy-preview-180--docs-relational-migrator.netlify.app/jobs/stopping-jobs/)
[Recover a Migration Job](https://deploy-preview-180--docs-relational-migrator.netlify.app/jobs/recover-jobs/)
[Pause and Resume a Migration Job](https://deploy-preview-180--docs-relational-migrator.netlify.app/jobs/pause-resume-kafka-jobs/)

## JIRA

https://jira.mongodb.org/browse/DOCSP-45120

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Is this free of spelling errors?
- [x] Is this free of grammatical errors?
- [x] Is this free of staging / rendering issues?
- [x] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

**Review by jmd-mongo — COMMENTED** [review]
Thanks for these updates, @ltran-mdb2! Overall, this looks good and the replacements make sense. 

One question I have is about the `job` and `job_plural` constants defined in snooty.toml: should we use these replacements throughout? I don't see any cases where we're using `{+job+}` or `{+job_plural+}`. I think we should use these as much as makes sense, or consider removing them entirely.

Thanks,
Joe

**Comment by ltran-mdb2** [comment]
Thanks for your review @jmd-mongo ! Because the phrase "migration job" is used in so many forms throughout the docs (Migration Job, Migration Jobs, Migration job, migration job, migration jobs. etc), I removed the constants since there is not one form of the phrase that is overwhelmingly used.

**Review by jmd-mongo — APPROVED** [review]
Thanks, @ltran-mdb2! This one LGTM.

Thanks,
Joe

**Comment by ltran-mdb2** [comment]
Hi @tomhollander ! Would you be able to review this PR that changes all mentions of "sync job" to "migration job"?

File: source/jobs/sync-jobs.txt [hunk 1]
```diff
@@ -1,37 +1,37 @@
...
+==============
 
 .. contents:: On this page
    :local:
    :backlinks: none
    :depth: 1
    :class: singlecol
 
-Sync jobs transfer data from your relational database to MongoDB. Sync 
+Migration jobs transfer data from your relational database to MongoDB. Sync
```
Comment (tomhollander) [inline source/jobs/sync-jobs.txt:13]: Missed one.

**Review by tomhollander — CHANGES_REQUESTED** [review]
Missed one in a pretty crucial spot. LGTM otherwise.

**Review by tomhollander — APPROVED** [review]

**Pull request merged**